specification/web/resource-manager/Microsoft.Web/stable/2022-03-01/WebApps.json

```diff
@@ -22339,6 +22557,83 @@
         }
       }
     },
+    "CsmDeploymentStatus": {
+      "description": "Deployment status response payload.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "./CommonDefinitions.json#/definitions/ProxyOnlyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "CsmDeploymentStatus resource specific properties",
+          "type": "object",
+          "properties": {
+            "deploymentId": {
+              "description": "Deployment operation id..",
+              "type": "string"
+            },
+            "status": {
+              "description": "Deployment build status.",
+              "type": "string"
```

Comment:
should it be an enum (which include succeeded, failed, etc.) with model-as-string=false?